.github/workflows/pull_request_check_action.yml

```diff
@@ -48,10 +48,8 @@ jobs:
 
       - name: Setup the database before running migrations
         run: |
-          apt-get update
```

Comment:
Removing `apt-get update` breaks the next line because `apt` doesn't find `postgresql-client` in its local registry.

Reply:
Okay, let me fix that then